include/swift/SIL/SILInstruction.h

```diff
@@ -7969,9 +7969,11 @@ class DifferentiableFunctionExtractInst
           SingleValueInstruction> {
 private:
   /// The extractee.
-  NormalDifferentiableFunctionTypeComponent extractee;
+  NormalDifferentiableFunctionTypeComponent Extractee;
   /// The list containing the `@differentiable` function operand.
-  FixedOperandList<1> operands;
+  FixedOperandList<1> Operands;
+  /// True if the instruction was created with an explicit type.
```

Comment:
I think that comments on a property in an instruction definition should describe what the property means to the identity of the instruction, not how it relates to "creation" or any particular action.

Reply:
Fixed in https://github.com/apple/swift/pull/27923/commits/da3e68c5b72ec574683d9856c6561e861900c9b3.
New doc comment:
```
/// True if the instruction has an explicit extractee type.
```